jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -99,7 +100,7 @@
             ArrayList.class, HashSet.class, Queue.class, List.class, Set.class, Collection.class
         };
     private static final Iterable<ConfigurationProcessorFactory> __factoryLoader = ServiceLoader.load(ConfigurationProcessorFactory.class);
-    private static final XmlParser __parser = initParser();
+    private static final Pool<ConfigurationParser> __parsers = new Pool<>(Runtime.getRuntime().availableProcessors(),1);
```

Comment:
I would cap that `Runtime.getRuntime().availableProcessors()` up to a reasonable number (8? 16? 32?) as I don't think we could use 128+ parsers (which is becoming a more and more common HW thread count) in parallel in practice. Or could we?

Reply:
Good idea.  Considering we were previously contending on 1 instance, I think a limit of 8 would be fine.